app/views/categories/_form.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= form_with(model: category) do |form| %>
+  <div class="category-form">
+    <div class="field">
+      <%= form.text_field :name, placeholder: "Name", class: 'category-input', required: true %>
+    </div>
+
+    <div class="field">
+      <%= form.text_field :icon, placeholder: "Image URL", class: "category-input", required: true %>
+    </div>
+
```

Comment:
- [ ] Kindly validate this field to accept only valid URL. when I enter random text like `dss` the application will be broken.
![Screenshot from 2022-03-25 14-37-15](https://user-images.githubusercontent.com/65860678/160123200-5735bc9f-63a8-4ea3-8954-f7f876ef589e.png)

Reply:
Thank you @uwadona. Validation added